### ISSUE TITLE
Migrate to faster web server

### ISSUE THREAD

**Issue by magicspacekiwi** [artscilab/admin#72]
This issue is for discussion involving finding suitable new hosting for the ArtSciLab servers which host the lab's website, the ARTECA staging environment, Creative Disturbance, and some other projects.

I will be evaluating some potential options over the coming weeks.

Currently, I'm only evaluating options that meet the following criteria
_(if you have input regarding this list, please leave a reply)_

≥ 2GB Memory
≥ 40GB Disk Space
VPS or Dedicated Server

//cc @cassininazir @duncangallagher

**Comment by cassininazir** [comment]
Perhaps another way of looking at this is removing things that are taking up processing power unnecessarily. @colin-c maybe we can take a look together this week.

**Comment by rmalina** [comment]
cassini
do we know if there is the same problem on arteca

solve it first on arteca and move the podcasts in question to arteca
first

and then later we can address the cd issu

we need to focus on getting podcasts up regularly on arteca

roger

Roger F Malina
is in Dallas +15108532007  malina.diatrope.com

**Comment by magicspacekiwi** [comment]
@rmalina The production ARTECA site ([arteca.mit.edu](http://arteca.mit.edu)) doesn't suffer from the performance issues we have on the staging site ([arteca.atec.io](http://arteca.atec.io/)), which is what this issue is talking about.

I just tried it out through a Sydney-based VPN and podcasts worked fine on the MIT site.

**Comment by magicspacekiwi** [comment]
@cassininazir I noticed that Apache SpamAssassin was running on the machine, but we don't need to since we don't use it to receive email so I disabled it. This doesn't appear to have made a significant difference in performance, and I don't really see a lot of other junk processes.

I monitored server resource usage when loading CPanel, the homepage of Creative Disturbance, and the homepage of ARTECA. Any of these overloaded the CPU (100% usage).

Additionally, disk performance is highly variable but typically pretty abysmal (~25MB/s on average) in my tests.

Another area of concern is that the system is out of date, running a release of CentOS from May 2016 and PHP from April 2016. Unfortunately system software is managed by HostGator.


---

In conclusion, I believe this server isn't adequate for our needs.

**Comment by magicspacekiwi** [comment]
@cassininazir Here are the screenshots you requested.

#### Showing 100% CPU utilization any time PHP is doing work
![cpu-usage](https://cloud.githubusercontent.com/assets/5863768/23222371/40ae44d0-f8ed-11e6-8264-d8b1ccd2e8ef.png)

_Note that the multiple mysqld processes are actually child processes - I was sorting by CPU utilization instead of using tree view_

#### Showing poor disk performance (30MB/s)
![disk-performance](https://cloud.githubusercontent.com/assets/5863768/23222370/40adb542-f8ed-11e6-9f7b-93d367c7d07a.png)

_In addition, it performs unbelievably slow when running the same test using /dev/random which makes me think the processor is **very** underpowered_

**Comment by magicspacekiwi** [comment]
We've started to migrate ArtSciLab sites to a new DigitalOcean machine which is faster.

I'm curious to know if this solves the issue of Creative Disturbance access from other countries (https://github.com/creativedisturbance/podcasts/issues/655)
